modules/cnf-topology-aware-lifecycle-manager-operator-update.adoc

```diff
@@ -53,17 +53,54 @@ spec:
 <1> The index image URL contains the desired Operator images. If the index images are always pushed to the same image name and tag, this change is not needed.
 <2> Set how frequently the Operator Lifecycle Manager (OLM) polls the index image for new Operator versions with the `registryPoll.interval` field. This change is not needed if a new index image tag is always pushed for y-stream and z-stream Operator updates. The `registryPoll.interval` field can be set to a shorter interval to expedite the update, however shorter intervals increase computational load. To counteract this, you can restore `registryPoll.interval` to the default value once the update is complete.
 <3> Last observed state of the catalog connection. The `READY` value ensures that the `CatalogSource` policy is ready, indicating that the index pod is pulled and is running. This way, {cgu-operator} upgrades the Operators based on up-to-date policy compliance states.
++
+[NOTE]
+====
+After a catalog source update, it takes time for OLM to update the subscription status. This can mean that the status of the subscription policy may continue to show as compliant when TALM decides whether remediation is needed. As a result the Operator specified in the subscription policy does not get upgraded. To mitigate this issue, the `lastObservedState: READY` specification reduces the time between the completion of the catalog source policy remediation and the update of the subscription status. 
```

Comment:
"it" can be a vague referent, so maybe be more specific, such as  

> After a catalog source update, the OLM might take some time to update the subscription status.

Reply:
According to the SSG, [admonitions](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions) should be short and concise, so maybe an optional step that includes a note might work?

Reply:
May/might

![Screenshot from 2023-09-25 16-42-16](https://github.com/openshift/openshift-docs/assets/57954076/7ee879bb-730e-4eab-8d38-2f67b7e2ffb7)